backend/api/quivr_api/modules/knowledge/controller/knowledge_routes.py

```diff
@@ -39,6 +39,9 @@ async def list_knowledge_in_brain_endpoint(
     """
 
     validate_brain_authorization(brain_id=brain_id, user_id=current_user.id)
+    logger.debug(
```

Comment:
lowercase for logging.

We can remove this line as it was for internal debug not for production debug